### PR TITLE
Move email OTP pages to authentication portal

### PR DESCRIPTION
**Pull request by mefarazath**
Move email OTP pages to authentication portal

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/emailotp.jsp [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<!--
+ ~ Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (inthirakumaaran) [inline apps/authentication-portal/src/main/webapp/emailotp.jsp:2]:
```suggestion
 ~ Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

File: apps/authentication-portal/src/main/webapp/emailAddress.jsp [hunk 2]
```diff
@@ -0,0 +1,192 @@
+<%--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (inthirakumaaran) [inline apps/authentication-portal/src/main/webapp/emailAddress.jsp:2]:
```suggestion
  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties [hunk 3]
```diff
@@ -182,19 +182,35 @@ last.accessed=Dernier accès
 Updated.the.password.successfully=Mot de passe mis à jour avec succès
 resend.confirmation.page.title=Renvoyer un courriel de confirmation
 resend.confirmation.page.message=Veuillez compléter le captcha ci-dessous.
+error.retry=Authentification échouée ! Veuillez réessayer
+authenticate=S'authentifier
```
Comment (inthirakumaaran) [inline apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties:186]: Is this correct?

File: apps/authentication-portal/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -312,6 +312,21 @@
...
+    </servlet>
+
+    <servlet>
+        <servlet-name>email_otp.do</servlet-name>
+        <jsp-file>/emailotp.jsp</jsp-file>
+    </servlet>
+
+    <servlet>
+        <servlet-name>email_otp_error.do</servlet-name>
+        <jsp-file>/emailotpError.jsp</jsp-file>
```
Comment (pulasthi7) [inline apps/authentication-portal/src/main/webapp/WEB-INF/web.xml:327]:
```suggestion
        <jsp-file>/emailOtpError.jsp</jsp-file>
```
Will need to rename file

File: apps/authentication-portal/src/main/webapp/WEB-INF/web.xml [hunk 5]
```diff
@@ -312,6 +312,21 @@
...
     </servlet>
 
+    <servlet>
+        <servlet-name>email_otp_enroll.do</servlet-name>
+        <jsp-file>/emailAddress.jsp</jsp-file>
+    </servlet>
+
+    <servlet>
+        <servlet-name>email_otp.do</servlet-name>
+        <jsp-file>/emailotp.jsp</jsp-file>
```
Comment (pulasthi7) [inline apps/authentication-portal/src/main/webapp/WEB-INF/web.xml:322]:
```suggestion
        <jsp-file>/emailOtp.jsp</jsp-file>
```

File: apps/authentication-portal/src/main/webapp/emailAddress.jsp (hunk 2 above)
Comment (IsuraD) [inline apps/authentication-portal/src/main/webapp/emailAddress.jsp:2]: 2021

File: apps/authentication-portal/src/main/webapp/emailotp.jsp (hunk 1 above)
Comment (IsuraD) [inline apps/authentication-portal/src/main/webapp/emailotp.jsp:2]: 2021

File: apps/authentication-portal/src/main/webapp/emailAddress.jsp [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  --%>
+
+<%@ page import="org.owasp.encoder.Encode"%>
```
Comment (IsuraD) [inline apps/authentication-portal/src/main/webapp/emailAddress.jsp:19]: This page is not formatted.

File: apps/authentication-portal/src/main/webapp/emailAddress.jsp (hunk 2 above)
Comment (mefarazath) [reply apps/authentication-portal/src/main/webapp/emailAddress.jsp:2]: These are existing pages coming from the email otp endpoint. Is it fine to change them to 2021?

**Review by IsuraD — APPROVED** [review]

**Pull request merged**